skopt/tests/test_searchcv.py

```diff
@@ -314,3 +317,21 @@ def test_searchcv_total_iterations():
     )
 
     assert opt.total_iterations == 10 + 5
+
+
+def test_search_cv_internal_parameter_types():
+    # Test whether the parameters passed to the
+    # estimator of the BayesSearchCV are of standard python
+    # types - float, int, str
+
+    # Below is example code that used to not work.
+    X = np.random.randn(10, 1)
+    y = X[:, 0] > 0
+
+    model = BayesSearchCV(
+        estimator=SVC(),
```

Comment:
Instead of `SVC` can use use our own classifier class that checks the types of the parameters?

Reply:
:+1: